Triplehelix490/experimental-light/experimental-light.ino

```diff
@@ -0,0 +1,47 @@
+// the elements which create the charicteristics of the code \/\/\/\/\/\/
+int pin = 9;
+int brightness = 0;
+int fadeAmount = 5;
+int time = 100;
+int flash = 3;
+
+// the setup pre-program \/
```

Comment:
I spoke to Richard and he said you used \/ to close your comment but // comments out only that one line.
If you use /* then it will comment out everything until you close it with */